client/client-multi/client-c/src/kaa/platform-impl/common/kaa_client_properties.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2014-2016 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <platform/kaa_client_properties.h>
+#include <platform/stdio.h>
+#include <platform/mem.h>
+#include "utilities/kaa_mem.h"
+
+#define KAA_DEFAULT_WORKING_DIRECTORY "\0"
```

Comment:
Furthermore, I consider the `working_directory` an incorrect name for the variable and constant. This is just a path to a directory where the Kaa client performs file I/O. You'd better rename the constant to `KAA_DEFAULT_STORAGE_DIRECTORY` and the variable to `storage_directory`.